keel-echo/src/main/kotlin/com/netflix/spinnaker/keel/echo/UnhealthyNotificationListener.kt

```diff
@@ -64,9 +82,9 @@ class UnhealthyNotificationListener(
 
     return Notification(
       subject = "${resource.spec.displayName} is unhealthy",
-      body = "<$resourceUrl|${resource.id}> is unhealthy and " +
-        "Spinnaker does not detect any changes to this resource. " +
-        "Manual intervention might be required. Please check the History view to see more details.",
+      body = "<$resourceUrl|${resource.id}> has been unhealthy for ${friendlyDuration(unhealthyDuration)} and " +
+        "Spinnaker cannot fix it. " +
```

Comment:
I like this wording much better.

Reply:
nit, I tend to recommend contractions wherever possible in UI/UX copy because they read more naturally vs. as 'formal' communications

```suggestion
      body = "<$resourceUrl|${resource.id}> has been unhealthy for ${friendlyDuration(unhealthyDuration)} and " +
        "Spinnaker can't fix it. " +
```

Reply:
Oh good to know. Y'all keep saying I'm not professional in my wording, apparently I over-indexed on that :D